specs/protocol/isthmus/derivation.md

```diff
@@ -336,7 +337,18 @@ The [EIP-7702] transaction format is as follows.
 - `access_list`: The [EIP-2930] access list.
 - `authorization_list`: The [EIP-7702] signed authorization list.
 
-Span batches with transaction type `4` should only be accepted after Isthmus is enabled.
+## Activation
+
+Singular batches with transactions of type `4` must only be accepted if Isthmus is active at the
+timestamp of the batch. If a singular batch contains a transaction of type `4` before Isthmus is
+active, this batch must be _dropped_. Note that according to Holocene derivation, this will also
+lead to the remaining span batch, and channel that contained it, to get dropped.
```

Comment:
Clarifying here that this means _pre_ holocene if a batch within a span batch is invalidated by an EIP-7702 transaction, the whole span batch is dropped correct?

Reply:
Nevermind - clarified below.

Reply:
Yes that's correct. If a pre-Holocene L2 chain uses a Pectra-activated L1 chain, a single SetCode tx in a span batch will lead to dropping the whole span batch, in particular older batches.

We could technically also add a sentence about batch validity to the bedrock derivation specs, because this is really a retrofitted pre-Isthmus rule.

Reply:
Great thanks - that's how i interpreted it so I think it's clear as is. Approved :)